abis_mapping/templates/survey_site_visit_data_v2/mapping.py

```diff
@@ -124,7 +128,13 @@ def extract_temporal_defaults(
                 # Determine if any dates are present in the row
                 start_date: types.temporal.Timestamp = row["siteVisitStart"]
                 end_date: types.temporal.Timestamp = row["siteVisitEnd"]
-                site_visit_id: str = row["siteVisitID"]
+                site_visit_id = _string_or_none.validate_python(row["siteVisitID"])
```

Comment:
The output of the [frictionless string field](https://github.com/frictionlessdata/frictionless-py/blob/main/frictionless/fields/string.py#L87) is `str | None` .  Do we need to validate again?